server/routes/bookRoute.js

```diff
@@ -12,4 +12,6 @@ bookRoute.delete('/:id', Validate.id, BookController.deleteBook);
 bookRoute.post('/request', 
 Authenticate.isLoggedIn, Validate.requestBook, BookController.requestBook)
 
+bookRoute.post('/borrow', Authenticate.isLoggedIn, Authenticate.isAdmin, Validate.lendBook, BookController.lendBook);
```

Comment:
Is this only available to admins? What about super admins?

Reply:
I implemented isAdmin to check for both admin and super admin so the route is open to both of these people.